phalcon/mvc/collection.zep

```diff
@@ -1649,7 +1649,7 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
 	/**
 	 * Unserializes the object from a serialized string
 	 */
-	public function unserialize(string! data)
+	public function unserialize(data)
```

Comment:
The `data` must be always a string: http://php.net/manual/en/function.unserialize.php

Reply:
hmm... it looks fine on paper ... i will double check this tomorrow cuz i remember it was not working